### PR TITLE
Books list

### PR DESCRIPTION
**Pull request by bhflm**
## Summary

Added `books-list.html`

## Screenshots

vucs

## Trello Card

https://trello.com/c/7oyagsmG/36-listado-de-libros

### CODE REVIEW

File: maquetado/books-list.html [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Books list</title>
+  <meta name="description" content="books-list">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <div class="big-container">
```
Comment (Dorian30) [inline maquetado/books-list.html]: I would change this class name for something more semantic: 

```suggestion
  <div class="books-list-container">
```

File: maquetado/books-list.html [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    <div class="books-list-container">
```
Comment (Dorian30) [inline maquetado/books-list.html]: Then we could write this instead

```suggestion
    <div class="books-list">
```

File: maquetado/books-list.html [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+      <div class="book-preview">
+        <img src ="./assets/book-cover.png" class="book-small-preview">
+        <h1 class="book-preview-info bold">Titulo del libro</h1>
+        <h1 class="book-preview-info detail-color">Autor del libro</h1>
```
Comment (Dorian30) [inline maquetado/books-list.html]: I would change this html tag for `h2`

File: maquetado/books-list.html [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      </div>
+      <div class="book-preview">
+        <img src ="./assets/book-cover.png" class="book-small-preview">
+        <h1 class="book-preview-info bold">Titulo del libro</h1>
+        <h1 class="book-preview-info detail-color">Autor del libro</h1>
```
Comment (Dorian30) [inline maquetado/books-list.html]: Same. There should be only one `h1` tag per section (In this case). Ideally you should have only one `h1` tag to describe your current page and `h2` for posts and articles sections.

File: maquetado/books-list.html [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+      </div>
+      <div class="book-preview">
+        <img src ="./assets/book-cover.png" class="book-small-preview">
+        <h1 class="book-preview-info bold">Titulo del libro</h1>
```
Comment (Dorian30) [inline maquetado/books-list.html]: Same

File: maquetado/books-list.html [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
```
Comment (Dorian30) [inline maquetado/books-list.html]: Same

File: maquetado/books-list.html [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
```
Comment (Dorian30) [inline maquetado/books-list.html]: Same

File: maquetado/books-list.html [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
```
Comment (Dorian30) [inline maquetado/books-list.html]: Same

File: maquetado/books-list.html [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
```
Comment (Dorian30) [inline maquetado/books-list.html]: Same

File: maquetado/books-list.html [hunk 10]
```diff
@@ -0,0 +1,57 @@
...
```
Comment (Dorian30) [inline maquetado/books-list.html]: Same

File: maquetado/scss/styles.scss [hunk 11]
```diff
@@ -9,6 +9,45 @@
...
+  align-content: center;
+}
+
+.book-preview {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  margin: 10px;
+  padding: 30px 30px;
+  background-color: #FFF;
```
Comment (Dorian30) [inline maquetado/scss/styles.scss]: Set this as `$white` and remember to order them in alphabetical order.

File: maquetado/scss/styles.scss [hunk 12]
```diff
@@ -9,6 +9,45 @@
...
+  box-shadow: 4px 4px 10px 0 rgba(0, 0, 0, 0.5);
```
Comment (Dorian30) [inline maquetado/scss/styles.scss]: Set `$black-05` here

File: maquetado/scss/styles.scss [hunk 13]
```diff
@@ -9,6 +9,45 @@
...
+.books-list-container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+  align-content: center;
+}
+
+.book-preview {
+  display: flex;
+  flex-direction: column;
```
Comment (Dorian30) [inline maquetado/scss/styles.scss]: You can use `column` class instead in the html tags

File: maquetado/scss/styles.scss [hunk 14]
```diff
@@ -9,6 +9,45 @@
...
 }
 
+.big-container {
+  padding: 10px;
+}
+
+.books-list-container {
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
```
Comment (Dorian30) [inline maquetado/scss/styles.scss]: Same, use `column` class for this. This way you don't need to define `display: flex` again and again. Remember the alphabetical order

File: maquetado/scss/styles.scss [hunk 15]
```diff
@@ -9,6 +9,45 @@
...
+  width: 198px;
+}
+
+.book-preview-info {
+  margin-top: 10px;
+}
+
+.books-list-container {
+  display: flex;
+  flex-direction: row;
```
Comment (Dorian30) [inline maquetado/scss/styles.scss]: Use `row` class instead

File: maquetado/books-list.html (hunk 1 above)
Comment (bhflm) [reply maquetado/books-list.html]: oops this wasn't meant to be named like this 😅

File: maquetado/scss/styles.scss [hunk 16]
```diff
@@ -10,7 +10,42 @@
...
+  height: 300px;
+  width: 200px;
+}
+
+.book-preview-info {
+  margin-top: 10px;
+}
+
+.books-list-container {
+  display: flex;
```
Comment (Dorian30) [inline maquetado/scss/styles.scss]: Use `row` instead

File: maquetado/books-list.html (hunk 3 above)
Comment (damfinkel) [reply maquetado/books-list.html]: Only one h1 in the document at the same time: https://www.hobo-web.co.uk/headers/

File: maquetado/scss/styles.scss [hunk 17]
```diff
@@ -8,6 +8,50 @@
...
+  flex-wrap: wrap;
+  align-content: center;
+}
+
+.book-preview {
+  margin: 10px;
+  padding: 30px 30px;
+  background-color: $white;
+  box-shadow: 5px 10px $black-05;
```
Comment (damfinkel) [inline maquetado/scss/styles.scss]: Try `5px 6px 8px rgba(0, 0, 0, 0.4);`, it looks better. This lacks blur. Remind me to show you a trick with box-shadow later.

File: maquetado/books-list.html [hunk 18]
```diff
@@ -0,0 +1,57 @@
...
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Books list</title>
+  <meta name="description" content="books-list">
+  <meta name="author" content="Wolox">
+  <link rel="stylesheet"  type="text/css" href="application.css">
+</head>
+<body>
+  <div class="books-list-container">
```
Comment (damfinkel) [inline maquetado/books-list.html]: This div has only one child. This means it's not necessary

**Pull request merged**